wandb/sdk/wandb_settings.py

```diff
@@ -506,6 +512,14 @@ def sync_file(self) -> str:
     def files_dir(self) -> str:
         return self._path_convert(self.files_dir_spec)
 
+    @property
+    def tmp_dir(self) -> str:
+        return self._path_convert(self.tmp_dir_spec) or tempfile.gettempdir()
```

Comment:
@raubitsj This adds a `/tmp` directory to all runs.  I figured it could be useful for other staging we might need to do...  

The tests exposed that `_path_convert` can return `None`.  How do we make `mypy` complain about `_path_convert` returning `None`?